microbench/PR1977_bench.cpp

```
@@ -63,7 +63,7 @@ int main(int argc, char** argv) {
 			}
 			unsigned int k;
 			if (New_sum > 32768) {
-				k = rand() % New_sum;
+				k = fastrand() % New_sum;
 			} else {
 				k = fastrand() % New_sum;
```

Cppcheck (2.21.0)

[error] Division by zero.

(zerodiv)